mantle/kola/tests/misc/network.go

```diff
@@ -98,13 +98,12 @@ NextProcess:
 		portData := strings.Split(parts[4], ":")
 		port := portData[len(portData)-1]
 		processData := parts[len(parts)-1]
-		pidStr := regexp.MustCompile(`pid=\d+`).FindString(processData)  // pid has a  leading 'pid=' identifier
 		processStr := regexp.MustCompile(`".+"`).FindString(processData) // process name is captured inside double quotes
-		if pidStr == "" || processStr == "" {
-			c.Errorf("%v did not contain pid and program; full output: %q", processData, output)
+		if processStr == "" {
+			c.Errorf("%v did not contain program; full output: %q", processData, output)
 			continue
 		}
-		pid, process := pidStr[4:], processStr[1:len(processStr)-1]
+		process := processStr[1 : len(processStr)-1]
```

Comment:
```suggestion
		process := processStr[1:len(processStr)-1]
```

I don't think it's common to include white space here.

Reply:
I actually removed the whitespace originally, but it failed some CI test because it didn't comply with `gofmt` :(
For some reason `gofmt` says there should be spaces there, even though it also said there should be no spaces there when that line was `pid, process := pidStr[4:], processStr[1:len(processStr)-1]`

Reply:
hmm weird.. ok sounds like a good enough reason.